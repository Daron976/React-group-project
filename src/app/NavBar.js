import { Link } from 'react-router-dom';
import logo from '../resources/planet.png';

const NavBar = () => (
  <div className="navs">
    <div className="NavBar">
      <img src={logo} alt="logo" />
      <h1> Space Rockets</h1>
    </div>
    <div className="Links">
      <div
        className="link-container"
      >
        <Link to="/" className="home">Rockets</Link>
        <div className="divider">|</div>
        <Link to="Missions" className="missions">Missions</Link>
        <div className="divider">|</div>
        <Link to="MyProfile" className="myProfile">My Profile</Link>
      </div>
    </div>
  </div>
);
export default NavBar;
