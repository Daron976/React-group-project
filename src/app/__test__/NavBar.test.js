import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavBar from '../Navigation/NavBar';

describe('Navigation', () => {
  test('Navigation container items renders as expected', async () => {
    const snapNavigation = renderer.create(
      <BrowserRouter>
        <NavBar />
      </BrowserRouter>,
    )
      .toJSON();
    expect(snapNavigation).toMatchSnapshot();
  });
});
