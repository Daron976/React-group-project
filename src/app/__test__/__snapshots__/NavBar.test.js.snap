// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Navigation container items renders as expected 1`] = `
<div
  className="navs"
>
  <div
    className="NavBar"
  >
    <img
      alt="logo"
      src="planet.png"
    />
    <h1>
       Space Rockets
    </h1>
  </div>
  <div
    className="Links"
  >
    <div
      className="link-container"
    >
      <a
        className="home"
        href="/"
        onClick={[Function]}
      >
        Rockets
      </a>
      <div
        className="divider"
      >
        |
      </div>
      <a
        className="missions"
        href="/Missions"
        onClick={[Function]}
      >
        Missions
      </a>
      <div
        className="divider"
      >
        |
      </div>
      <a
        className="myProfile"
        href="/MyProfile"
        onClick={[Function]}
      >
        My Profile
      </a>
    </div>
  </div>
</div>
`;
